package.json

```diff
@@ -3,31 +3,36 @@
   "private": true,
   "version": "0.0.0",
   "type": "module",
+  "packageManager": "pnpm@7.13.4",
   "scripts": {
     "dev": "vite",
-    "build": "vite build",
+    "build": "tsc && vite build",
     "preview": "vite preview",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "storybook build"
   },
   "dependencies": {
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.18.13",
-    "@storybook/addon-actions": "^7.0.0-alpha.29",
-    "@storybook/addon-essentials": "^7.0.0-alpha.29",
-    "@storybook/addon-interactions": "^7.0.0-alpha.29",
-    "@storybook/addon-links": "^7.0.0-alpha.29",
-    "@storybook/builder-vite": "^7.0.0-alpha.29",
-    "@storybook/react-vite": "^7.0.0-alpha.29",
-    "@storybook/testing-library": "^0.0.14-next.0",
+    "@babel/core": "^7.19.3",
     "@types/react": "^18.0.17",
     "@types/react-dom": "^18.0.6",
-    "@vitejs/plugin-react": "^2.0.1",
+    "@vitejs/plugin-react": "^2.1.0",
     "babel-loader": "^8.2.5",
-    "storybook": "^7.0.0-alpha.29",
-    "vite": "^3.0.7"
+    "typescript": "^4.6.4",
+    "vite": "^3.1.0",
+    "webpack": "^5.74.0",
+    "storybook": "^7.0.0-alpha.35",
+    "@storybook/react": "^7.0.0-alpha.35",
+    "@storybook/react-vite": "^7.0.0-alpha.35",
+    "@storybook/addon-links": "^7.0.0-alpha.35",
+    "@storybook/addon-essentials": "^7.0.0-alpha.35",
+    "@storybook/addon-interactions": "^7.0.0-alpha.35",
+    "@storybook/testing-library": "^0.0.13"
+  },
+  "resolutions": {
+    "@mdx-js/react": "2.1.5"
```

Comment:
Let's remove this workaround, I'd like this to keep erroring, so that we remember we need to deal with it in storybook.